src/main/jhipster/src/main/webapp/app/shared/germplasm/service/germplasm.service.ts

```diff
@@ -71,20 +71,15 @@ export class GermplasmService {
             { params, observe: 'response' });
     }
 
+    // FIXME into IBP-4659 (Replace by Resource /variables/filter)
```

Comment:
this function getGermplasmAttributes can be removed 
@cuenyad @nahuel-soldevilla

Reply:
done